app/lib/dom.ts

```diff
@@ -0,0 +1,33 @@
+export function isInSubtreeOf(element: Element, target: Element) {
+  let current: Element | null = target;
+  while (current) {
+    if (current === element) return true;
+    current = current.parentElement;
+  }
+  return false;
+}
+
+export function setClassEnabled(element: HTMLElement, className: string, enabled: boolean) {
+  if (enabled) {
+    element.classList.add(className);
+  } else {
+    element.classList.remove(className);
+  }
+}
+
+export function setOrRemoveStyleProperty(
```

Comment:
seems like element.style.setProperty would suffice - if value isn't true it just sets it to an empty string

Reply:
Removed